doc/source/whatsnew/v0.24.0.txt

```diff
@@ -159,6 +159,30 @@ This is the same behavior as ``Series.values`` for categorical data. See
 :ref:`whatsnew_0240.api_breaking.interval_values` for more.
 
 
+.. _whatsnew_0240.enhancements.multi_index_repr:
+
+Better repr for MultiIndex
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Previously, outputting a :class:`MultiIndex` printed the levels/labels of the
```

Comment:
add the issues to this first paragraph at the end (I think you have only 1 here?)

Reply:
Yeah, only one issue, though another similar issue is closed also.

Adding the issue to the first paragraph looks out of place. I can rearrange the paragraphs instead.